charts/cortx/templates/_images.tpl

```diff
@@ -0,0 +1,14 @@
+{{/*
+Return a valid CORTX image name from parts
+{{ include "cortx.images.image" ( dict "image" .Values.path.to.the.image "root" $) }}
+*/}}
+{{- define "cortx.images.image" -}}
+{{- printf "%s/%s:%s" .image.registry .image.repository (default .root.Chart.AppVersion .image.tag) -}}
```

Comment:
This works as long as the version is identical to the image tag. If that changes, we'll just have to go back to setting the tag in values.yaml.